spec/rubocop/cop/style/legacy_hash_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Style::LegacyHash, :config do
+  it 'reports an offense for single-argument Hash[]' do
+    expect_offense(<<~RUBY)
+      Hash[ary]
+      ^^^^^^^^^ Prefer ary.to_h to Hash[ary].
+    RUBY
+
+    expect_correction(<<~RUBY)
+      ary.to_h
+    RUBY
+  end
+
+  it 'reports different offense for multi-argument Hash[]' do
+    expect_offense(<<~RUBY)
+      Hash[a, b, c, d]
+      ^^^^^^^^^^^^^^^^ Prefer literal hash to Hash[arg1, arg2, ...].
+    RUBY
+
+    expect_correction(<<~RUBY)
+      {a => b, c => d}
+    RUBY
+  end
+
+  it 'reports different offense for empty Hash[]' do
+    expect_offense(<<~RUBY)
+      Hash[]
+      ^^^^^^ Prefer literal hash to Hash[arg1, arg2, ...].
+    RUBY
+
+    expect_correction(<<~RUBY)
+      {}
+    RUBY
+  end
+
+  it 'does not tries to correct multi-argument Hash with odd number of arguments' do
```

Comment:
For wordings of these example descriptions, please refer: https://github.com/rubocop-hq/rubocop/blob/77db90e6fa0f45ea7cb0b6ec4bee23faf66a385a/lib/rubocop/cop/internal_affairs/example_description.rb

Reply:
I don't understand this suggestion, sorry. What exactly should be fixed here (except for "tries"→"try" :joy:)?

Reply:
Yeah after correcting "tries" I think it's fine. The internal affairs cop is just to make sure that we don't have descriptions that contradict the actual test.